bazel/get_workspace_status

```diff
@@ -45,8 +45,8 @@ git_branch=$(git rev-parse --abbrev-ref HEAD)
 echo "BUILD_SCM_BRANCH ${git_branch}"
 
 git_remote=$(git remote get-url origin)
-git_remote_code=$?
-if [[ git_remote_code == 0 ]]; then
+git_remote_exit_code=$?
```

Comment:
how about just checking if `git_remote` was set - shellcheck doesnt like checking the return code and i cant off hand think of a way of iffing and setting at the same time - however, i think if previous command exitted non 0 then `git_remote` would not be set - so i think you can `if [[ -n "$git_remote" ]]; then ...`

Reply:
for more assurance you could change to...

```bash
git_remote=$(git remote get-url origin || echo "")
```

Reply:
I like that idea, let me give it a go!